config/Shared/config_default-docker.dev.php

```diff
@@ -171,4 +171,5 @@
     'payment' => $messageBrokerTransport,
     'assets' => $messageBrokerTransport,
     'product' => $messageBrokerTransport,
+    'search' => $messageBrokerTransport,
```

Comment:
please check how it looks like in suite-nonsplit, there is only SQS_TRANSPORT now

Reply:
Removed